web/modules/custom/collection/collection.module

```diff
@@ -40,6 +42,21 @@ function collection_entity_bundle_field_info_alter(&$fields, EntityTypeInterface
 }
 
 /**
+ * Implements hook_views_pre_render().
+ */
+function collection_views_pre_view(ViewExecutable $view) {
```

Comment:
The view displays a list of collections but the view config file itself is located in the profile. This cache manipulation should be located there as well since it's only serving this particular page.